src/uproot/interpretation/objects.py

```diff
@@ -510,13 +510,25 @@ def _unravel_members(members):
 def _strided_awkward_form(awkward, classname, members, file, context):
     contents = {}
     for name, member in members:
-        if isinstance(member, AsStridedObjects):
-            cname = uproot.model.classname_decode(member._model.__name__)[0]
-            contents[name] = _strided_awkward_form(
-                awkward, cname, member._members, file, context
-            )
+        if not context["header"] and name in ("@num_bytes", "@instance_version"):
+            pass
+        elif not context["tobject_header"] and name in (
+            "@num_bytes",
+            "@instance_version",
+            "@fUniqueID",
+            "@fBits",
+            "@pidf",
+        ):
+            pass
         else:
-            contents[name] = uproot._util.awkward_form(member, file, context)
+            if isinstance(member, AsStridedObjects):
+                cname = uproot.model.classname_decode(member._model.__name__)[0]
+                contents[name] = _strided_awkward_form(
+                    awkward, cname, member._members, file, context
+                )
+            else:
+                contents[name] = uproot._util.awkward_form(member, file, context)
```

Comment:
> dedent this

You don't mean remove the indentation from this line (breaking syntax) or the `else` block (changing the semantics): do you mean to take the `if` (line 524) and `else` (line 529) and make them part of an `elif` chain with the outer `else` (line 523)?

That would be equivalent, though I wrote it this way because I was mentally thinking of the two `if` blocks as different concerns. I write a lot of `elif` chains, but usually an `elif` chain is selecting among items that are all in the same category.

I don't have strong feelings about this one; if neither of us do, let's leave it as-is. Otherwise, make a suggestion and I'll commit it.